File: app/src/main/java/com/yuyu/module/utils/TabVO.java
package com.yuyu.module.utils;

import android.app.Fragment;

import lombok.Data;

@Data
public class TabVO {

    private Fragment fragment;
    private String title;

    public TabVO(Fragment fragment, String title) {
        this.fragment = fragment;
        this.title = title;
    }

}

File: app/src/main/java/com/yuyu/module/utils/TabVO.kt
package com.yuyu.module.utils

import android.app.Fragment

data class TabVO(val fragment: Fragment? = null, val title: String? = null)

/*
package com.yuyu.module.utils;

import android.app.Fragment;

import lombok.Data;
import lombok.experimental.Accessors;

@Data
@Accessors(chain = true)
public class TabVO {

    private final Fragment fragment;
    private final String title;

}
*/
